tests/shouldwork/Signal/Ram/RMulti.hs

```diff
@@ -2,44 +2,52 @@
 --
 -- Test reading with a fast clock. Memory is initialized through the write port,
 -- but after that the write side is dormant.
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
```

Comment:
This shouldn't be needed.

Reply:
Depends on your definition of "needed". I like to compile HDL with `-Wall` and not have any warnings, and then it's needed, since we use `createDomain` below.

So instead of "this shouldn't be needed", I'd like to counter "we should change CI so this actually is needed" 😆